src/main/java/com/zufar/onlinestore/cart/entity/ShoppingSession.java

```diff
@@ -0,0 +1,84 @@
+package com.zufar.onlinestore.cart.entity;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "shopping_session")
+public class ShoppingSession {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @Column(name = "user_id", nullable = false)
+    private UUID userId;
+
+    @OneToMany(mappedBy = "shoppingSession", cascade = CascadeType.ALL)
+    private Collection<ShoppingSessionItem> items;
```

Comment:
It is better to use List ot Set for entity

Reply:
Fixed

but why it is better?